caste.go

```diff
@@ -1474,3 +1493,11 @@ func trimZeroDecimal(s string) string {
 	}
 	return s
 }
+
+// Helper: Abs returns the absolute value of x.
+func Abs(x int) int {
```

Comment:
do we use this function anywhere? one thing to be careful, `int` is a specific type and not a generic numerical type, so calling this function will return an `int` and not preserve the type of the input

https://stackoverflow.com/questions/67678331/how-to-write-a-generic-function-that-accepts-any-numerical-type we could do something like this to enforce the output is the same type as the input

Reply:
Good catch. Removed `Abs` since we're not using it anymore anyway